pom.xml

```diff
@@ -2606,6 +2606,28 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>3.1.3</version>
+        <configuration>
+          <classFilesDirectory>${basedir}/target/scala-2.11/classes</classFilesDirectory>
```

Comment:
We may also want to apply it to 2.12 later?

Reply:
yeah, me too. Good catch.